change-validation/src/main/resources/change-validation.conf

```diff
@@ -0,0 +1,54 @@
+
+# Email server configuration
+# Global Infos
+smtp.hostServer=smtp.gmail.com
+smtp.port=587
+smtp.auth=true
+smtp.email="bonoblow@gmail.com"
+smtp.login="bonoblow@gmail.com"
+smtp.password="D;=$CvEwd=6:Y`aV[zI@%RE'x%4d>q"
```

Comment:
please don't commit actual data: 
- no password (change that one on the server), 
- no real host/email/login 

(same below for `efu@rudder.io` etc